### DISCUSSION TITLE
Invalid YAML breaks issue creation

### DISCUSSION THREAD

**Discussion by JustinBack** [community/community#21631] [Projects and Issues]
<p>Just noticed today that when submitting a config.yml, which is missing the <code>contact_links:</code> key, breaks the issue submission process.</p>
<pre><code class="lang-auto"> 
  - name: API
    url: https://tosdr.atlassian.net/jira/software/c/projects/API/issues/
    about: Report bugs or suggest features for the API.
</code></pre>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/4/84cffbbf374d295715fb6abb3adf6cd92f2571f6.jpeg" data-download-href="/uploads/short-url/iWUCXL9WPOSl6USaakjJavqYdg2.jpeg?dl=1" title="" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/40178814/181065905-3f6cd2f2-fdbc-4f3c-9615-96e4b3dbbd63.jpeg" alt="" data-base62-sha1="iWUCXL9WPOSl6USaakjJavqYdg2" width="690" height="349" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/4/84cffbbf374d295715fb6abb3adf6cd92f2571f6_2_690x349.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/8/4/84cffbbf374d295715fb6abb3adf6cd92f2571f6_2_1035x523.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/8/4/84cffbbf374d295715fb6abb3adf6cd92f2571f6.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/4/84cffbbf374d295715fb6abb3adf6cd92f2571f6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename"></span><span class="informations">1378×697 165 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/tosdr/CrispCMS/issues/new/choose">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/tosdr/CrispCMS/issues/new/choose" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/40178814/181065907-3a48e23e-ece0-443f-864a-e7568a11102a.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://github.com/tosdr/CrispCMS/issues/new/choose" target="_blank" rel="noopener nofollow ugc">Build software better, together</a></h3>

  <p>GitHub is where people build software. More than 65 million people use GitHub to discover, fork, and contribute to over 200 million projects.</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>The link above will soon work again as its our production repo :slight_smile:</p>

**Comment by JustinBack** [comment]
<p>I have created a test repository for this case:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/JustinBack/yaml-test/issues/new/choose" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:500/500;"><img src="https://user-images.githubusercontent.com/40178814/181065909-066b868a-4319-47f1-87cb-450e6a3bf8c0.png" class="thumbnail" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png"></div>

<h3><a href="https://github.com/JustinBack/yaml-test/issues/new/choose" target="_blank" rel="noopener nofollow ugc">Build software better, together</a></h3>

<p><a href="https://github.com/JustinBack/yaml-test/issues/new/choose" target="_blank" rel="noopener nofollow ugc">nBack/yaml-test/issues/new/choose</a></p>

  <p><span class="label1">GitHub is where people build software. More than 65 million people use GitHub to discover, fork, and contribute to over 200 million projects.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by canuckjacq** [comment]
<p>:wave: <a class="mention" href="/justinback">@JustinBack</a></p>
<p>Thanks for the report! I’ve opened an issue and I’ll do my best to keep you updated.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Confirming this has been fixed as per:</p>
<aside class="onebox githubissue">
  <header class="source">
      <a href="https://github.com/JustinBack/yaml-test/issues/1" target="_blank" rel="noopener">github.com/JustinBack/yaml-test</a>
  </header>
  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Issue">
	  <svg width="60" height="60" class="github-icon" viewbox="0 0 14 16" aria-hidden="true"><path d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"></path></svg>
  </div>

  <div class="github-info-container">
    <h4>
      <a href="https://github.com/JustinBack/yaml-test/issues/1" target="_blank" rel="noopener">Verifying the bug fix.</a>
    </h4>

    <div class="github-info">
      <div class="date">
        opened <span class="discourse-local-date" data-format="ll" data-date="2021-05-06" data-time="09:11:24" data-timezone="UTC">09:11AM - 06 May 21 UTC</span>
      </div>


      <div class="user">
        <a href="https://github.com/octatone" target="_blank" rel="noopener">
          <img alt="octatone" src="https://user-images.githubusercontent.com/11161530/181065912-d5b80f50-1628-4bbf-9c84-54e4fbeade12.png" class="onebox-avatar-inline" width="20" height="20">
          octatone
        </a>
      </div>
    </div>

    <div class="labels">
    </div>
  </div>
</div>

  <div class="github-row">
    <p class="github-body-container">About to ship a fix for this :)</p>
  </div>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
